packages/react-components/package.json

```diff
@@ -25,12 +25,18 @@
     "svgs": "^4.1.0"
   },
   "devDependencies": {
+    "@celo/eslint-config-typescript": "^0.0.1",
     "@celo/flake-tracker": "0.0.1-dev",
-    "@celo/typescript": "0.0.1",
     "@types/hoist-non-react-statics": "^3.3.1",
     "@types/react": "^16.9.34",
     "@types/react-native": "^0.63.37",
     "@types/react-native-autocomplete-input": "^4.0.1",
+    "@typescript-eslint/eslint-plugin": "^4.26.0",
+    "eslint": "^7.27.0",
+    "eslint-plugin-import": "^2.23.4",
+    "eslint-plugin-react": "^7.24.0",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "eslint-plugin-react-native": "^3.11.0",
```

Comment:
ugh, eslint plugins make it really annoying because eslint resolves plugins based on the location of the .eslintrc file.

So, one option is to create a "`eslint-celo`" eslint wrapper script that is included in the `@celo/eslint-config-typescript` module and uses the .eslintrc file included with  `@celo/eslint-config-typescript`.  The downside is that as soon you pass another config file, you'll need to install the peerDependencies, which might be unexpected for many users.

I'll think more about it.

Reply:
Another option is what's described [here](https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/MONOREPO.md#one-tsconfigjson-per-package-and-an-optional-one-in-the-root).